backend/src/test/java/com/redhat/cloud/notifications/processors/email/EmailSubscriptionTypeProcessorTest.java

```diff
@@ -89,6 +89,9 @@ void shouldSuccessfullySendEmail() {
         inMemoryConnector.source(AGGREGATION_CHANNEL).send(Json.encode(aggregationCommand1));
         inMemoryConnector.source(AGGREGATION_CHANNEL).send(Json.encode(aggregationCommand2));
 
+        // TODO Replace fixed delay with a dynamic one (wait until the message has been processed).
+        Thread.sleep(5000L);
```

Comment:
Could we use mutiny calls with retry.atMost for this?

Reply:
In this particular case, there's nothing we could test to interrupt the retry/wait.

But we could probably get rid of awaitability and replace it with Mutiny retries in other places.